test/clients/azure/client.go

```diff
@@ -22,9 +34,26 @@ func NewClientFromEnvironment() (*Client, error) {
 
 	subscriptionID := os.Getenv("AZURE_SUBSCRIPTION_ID")
 
+	var rpc externalapi.OpenShiftManagedClustersClient
```

Comment:
Can you move this part into its own PR and update UpdateOSACluster to use this client?

Reply:
It would certainly unblock my admin refactor and would speed up merging the scale test too.